sentry-android-core/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,6 @@
     <provider
       android:name=".SentryInitProvider"
       android:authorities="${applicationId}.SentryInitProvider"
-      android:exported="false"
-      android:initOrder="2147483647" />
+      android:exported="false" />
```

Comment:
Curious what happens if we don't define an order here?
We'd like our code to init early so we can capture error during initialization

Reply:
the system is gonna define the `android:initOrder` and it's gonna be before the `Application` anyway.
this seems to be more like a magic number